src/mrgw.sh

```diff
@@ -21,7 +22,7 @@ shift 2
 
 run_root=$script_root/run/$name
 pidfile=$run_root/out/radosgw.${port}.pid
-asokfile=$run_root/out/radosgw.${port}.asok
+asokfile=$($ceph_bin/ceph-conf --show-config-value admin_socket --name radosgw.${port})
```

Comment:
@cbodley sorry for the inconvenience. 

it was not a problem unless the `$run_root` is too long, so that the path name of asokfile exceeds the 107 limit. i was trying to find all the occurance of .asok, and to move them under `$TMPDIR`.